pkg/services/sqlstore/store.go

```diff
@@ -97,7 +97,6 @@ type Store interface {
 	SearchOrgUsers(ctx context.Context, query *models.SearchOrgUsersQuery) error
 	RemoveOrgUser(ctx context.Context, cmd *models.RemoveOrgUserCommand) error
 	GetDashboardTags(ctx context.Context, query *models.GetDashboardTagsQuery) error
-	SearchDashboards(ctx context.Context, query *models.FindPersistedDashboardsQuery) error
```

Comment:
interesting FindDashboards was not on the interface 😃, it is only used inside of sqlstore...

Reply:
I think we don't need it in the new service neither normally

Reply:
It wasn't in the original service interface, but `service.FindDashboards` is used: 
https://github.com/grafana/grafana/pull/49347/files#diff-286a50be4d4b92176e99c68873de717fa6f6755d83e6260f5f07fd5802742667L302

Reply:
i see, just never mocked in the test...